cmake/modules/Findlibuv.cmake

```diff
@@ -0,0 +1,87 @@
+find_package(PkgConfig)
```

Comment:
what is this file based on? Writing a find-module ourselves is a very tricky business so we should better document how we did come up with the contents of the file.

Reply:
Thanks for the suggestion! I created a new libuv find module based on https://github.com/Kitware/CMake/blob/master/Source/Modules/FindLibUV.cmake, and added a comment at the top of the file.